docs/reference/glossary-of-terms.md

```diff
@@ -85,6 +88,9 @@ A network of real-world entities (i.e., objects, events, situations, and concept
 #### Label
 Usually refers to a human-readable label corresponding to the `rdfs:label` [predicate](#predicate). Labels are typically unique per ontology. In OBO Format and in the bio-ontology literature, labels are sometimes called [Names](#Name). Sometimes in the machine learning literature, and in databases such as Neo4J, "label" actually refers to a [Category](#Category).
```

Comment:
Minor edits:
Usually refers to a human-readable text string corresponding to the `rdfs:label` [predicate](#predicate). Labels are typically unique per ontology. In OBO Format and in the bio-ontology literature, labels are sometimes called [Names](#Name). Sometimes in the machine learning literature and in databases such as Neo4J, "label" actually refers to a [Category](#Category).